package.json

```
@@ -44,12 +44,13 @@
     "devDependencies": {
         "@types/node": "^11.11.4",
         "ajv": "^6.10.0",
+        "github-changes": "^1.1.2",
         "ts-node": "^8.0.2",
         "tsc": "^1.20150623.0",
         "typescript": "^3.3.3"
     },
     "scripts": {
         "vscode:prepublish": "test -f ./syntaxes/Scala.tmLanguage.json",
-        "prepare": "npx ts-node src/typescript/GenerateTmLanguageFile.ts > ./syntaxes/Scala.tmLanguage.json"
+        "build": "npx ts-node src/typescript/GenerateTmLanguageFile.ts > ./syntaxes/Scala.tmLanguage.json"
```

using `npx` is great for one-off tasks, but it has two issues:

- it's slower since it has to re-download the module at every run.
- it downloads the latest version every time, which may break the build if future versions of the module have breaking changes

I would recommend adding ts-node to the dev-dependencies with

```
yarn add --dev ts-node
```

and doing

```suggestion
        "build": "ts-node src/typescript/GenerateTmLanguageFile.ts > ./syntaxes/Scala.tmLanguage.json"
```

here

---

Great catch! Updated.